en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md

```diff
@@ -7,6 +7,9 @@ Moreover, Choreo's Managed Authentication introduces the concept of a Backend Fo
 !!! note
     Managed Authentication is currently available only for React, Angular, and Vue.js buildpacks.
 
+!!! warn
+    Managed Authentication relies on "SAMESITE" cookie attribute for security. It is not recommended to be used with older browsers which do not have "SAMESITE" cookie support.
```

Comment:
- Why do we say so ("Managed Authentication" feature replying on cookies for maintaining user session. Such interactions require CSRF protection. In this context, SAMESITE cookie attribute is used to achieve CSRF protection).
- Are we going to provide instructions on what should be done if developer needs more security (reading non-http-only cookie and sending it in as a parameter, etc.). If CSRF protection is required for older browsers, should they not use "Managed Authentication" at all? 
- Link SAMESITE to https://owasp.org/www-community/SameSite
-

Reply:
Enables Managed Authentication with CSRF protection using the SAMESITE cookie attribute. Recommended for use with modern browsers that support SAMESITE

Reply:
Resolved with [79dea23](https://github.com/wso2/docs-choreo-dev/pull/1183/commits/79dea23b581488d180f2f7b43f80c45a33d6b236)